integration/flux_monitor/fixtures/job.json

```diff
@@ -12,6 +12,7 @@
         },
         "feeds": ["http://external-adapter:6644"],
         "threshold": 5,
+        "pollingInterval": "5s",
```

Comment:
Why not 1s?

Reply:
The minimum interval was set to the default HTTP timeout used by the Flux Monitor